File: Hoteles-Reserva-WS/src/java/models/EventAttendance.java
package models;

import java.util.List;

public class EventAttendance {
    private Events event;
    private List<Client> lstClient;

    public Events getEvent() {
        return event;
    }

    public void setEvent(Events event) {
        this.event = event;
    }

    public List<Client> getLstClient() {
        return lstClient;
    }

    public void setLstClient(List<Client> lstClient) {
        this.lstClient = lstClient;
    }
    
    
}

File: Hotel-Reservacion/Hotel-Reservacion/Models/EventAttendance.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace Hotel_Reservacion.Models
{
    public class EventAttendance
    {
        private Events events;
        private Client client;

        public Events Events
        {
            get
            {
                return events;
            }
            set
            {
                events = value;
            }
        }
        public Client Client
        {
            get
            {
                return client;
            }
            set
            {
                client = value;
            }
        }
    }
}